_widget/src/components/app/component.vue

```diff
@@ -53,6 +57,15 @@ export default {
     search: {
       type: Object,
       default() { return new Search(); }
+    },
+    sources: {
+      type: Array,
+      default() {
+        return [
+          { name: 'DNSimple Support', url: 'https://support.dnsimple.com' },
+          { name: 'DNSimple Developer', url: 'https://deploy-preview-760--dnsimple-developer.netlify.app' }
```

Comment:
Right now, it doesn't support CORS yet until https://github.com/dnsimple/dnsimple-developer/pull/760:

```suggestion
          { name: 'DNSimple Developer', url: 'https://developer.dnsimple.com' }
```